src/styles/_catalog.scss

```diff
@@ -1,22 +1,36 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$gap-size: 48px;
+$card-size: 200px;
+$container-width-488: $card-size*2 + $gap-size*1;
+$container-width-768: $card-size*3 + $gap-size*2;
+$container-width-1024: $card-size*4 + $gap-size*3;
 
-  max-width: 944px;
+.catalog {
+  padding-bottom: $gap-size;
+  display: grid;
+  gap: $gap-size;
+  grid-template-columns: repeat(auto-fill, $card-size);
+  
+  max-width: $card-size;
   margin: 0 auto;
 
+  @media (min-width: 488px) {
+    max-width: $container-width-488;
```

Comment:
you need to change grid-template-columns: repeat(numberOfItems, $card-size)
check all media